docs/src/pages/developers/technical-faq/index.mdx

```diff
@@ -121,15 +121,7 @@ If you're encountering this issue in production mode, try to check the worker lo
 
 ## How can I make imports run faster?
 
-The included `docker-compose` configuration for production mode has no understanding of how powerful your server is. It will run just a single instance of the services involved in data imports, `worker` , `ingest-file` and `convert-document`.
```

Comment:
I'd leave this section in, because it still holds true. Just the reference to `convert-document` needs to go.

Reply:
Left it back in!